magefiles/magefile.go

```diff
@@ -262,7 +262,7 @@ func (ci CI) TestE2E() error {
 }
 
 func RunE2ETests() error {
-	labelFilter := utils.GetEnv("E2E_TEST_SUITE_LABEL", "!upgrade-create && !upgrade-verify && !upgrade-cleanup && !release-pipelines")
+	labelFilter := utils.GetEnv("E2E_TEST_SUITE_LABEL", "!upgrade-create && !upgrade-verify && !upgrade-cleanup && !release-pipelines && !release-service ")
```

Comment:
This doesn't seem right. We want the `release-service` suite to run on CI. Just not the `release-pipelines` suite

Reply:
I saw this [All tests must have the label release-pipelines to avoid running them against the dev environment by OpenShift CI](https://github.com/redhat-appstudio/e2e-tests/tree/main/tests/release/pipelines#all-tests-must-have-the-label-release-pipelines-to-avoid-running-them-against-the-dev-environment-by-openshift-ci), so I also add release-service here to aviod the same situation. Correct me, if I am wrong. @johnbieren

Reply:
That is for the release-pipelines suite. This is the release-service suite. We want the release-service suite to run.. it is taking the place of the release suite